RELEASE.md

```diff
@@ -0,0 +1,84 @@
+# Release Instructions.
+
+> Note: Versions should be semantic versioning maj.min.patch style, except for the default
+> HEAD-SNAPSHOT which is the version used on the `main` branch
+
+Preconditions:
+
+  1. You must have a username on oss.sonatype.org.
+    - If you need one, sign up at issues.sonatype.org.
```

Comment:
always thought it was funny you had to sign up for an account via a JIRA ticket

Reply:
***twitches***